Include bluetooth/mgmt.h in bt-monitor compile test to make it fail
with BlueZ 5.x.

--- a/configure.in
+++ b/configure.in
@@ -1519,6 +1519,7 @@
 [
 #include <bluetooth/bluetooth.h>
 #include <bluetooth/hci.h>
+#include <bluetooth/mgmt.h>
 ],
 			[u_int i = HCI_CHANNEL_MONITOR],
 			ac_cv_lbl_hci_channel_monitor_is_defined=yes,
